### DISCUSSION TITLE
启动报错。数据库命令链接正常。

### DISCUSSION THREAD

**Discussion by juzhijian** [kbengine/kbengine#1553] [Q&A]
Logging to: D:\kbengine-2.5.12\server_assets/logs/dbmgr.*.log

KBE_ROOT = D:/kbengine-2.5.12/
KBE_RES_PATH = D:/kbengine-2.5.12/kbe/res/;D:/kbengine-2.5.12/server_assets/;D:/kbengine-2.5.12/server_assets/scripts/;D:/kbengine-2.5.12/server_assets/res/
KBE_BIN_PATH = D:/kbengine-2.5.12/kbe/bin/server/

server-configs:
        internalTcpAddr : 192.168.87.1:52716
        componentID : 4000

Logging(changed) to: D:\kbengine-2.5.12\server_assets/logs/logger_dbmgr.*.log

---- dbmgr Version: 2.5.10. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CEEA1830B25FC811C6B0172173A5FCD0. Config: Debug X64. Built: 17:49:58 Feb 27 2022. AppID: 4000. UID: 19399. PID: 17712 ----
Server MECHREVO: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz with 16631656kB RAM

[INFO]: TelnetServer server is running on port 32000
     dbmgr-1: [ERROR]: DBInterfaceMysql::attach: mysql_errno=2026, mysql_error=SSL connection error: unknown error number
     dbmgr-1: [ERROR]: DBUtil::createInterface: attach to database failed!
        dbinterface=0x40ed7ee768
        args=interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=no.

     dbmgr-1: [ERROR]: Dbmgr::initDB(): can't create dbInterface!
     dbmgr-1: [ERROR]: app::initialize(): initialization failed!

**Comment by huhubis** [comment]
网上查啊 mysql_errno=2026, mysql_error=SSL connection error: unknown error number
